### PR TITLE
Added Ojas Patel rough draft for dropout review

### PR DESCRIPTION
**Pull request by patelojas**

### CODE REVIEW

File: _summaries/srivastava2014dropout_1.md [hunk 1]
```diff
@@ -1,19 +1,114 @@
...
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+# Main Idea
+
+This paper looks to solve the problem of overfitting for large networks. It introduces dropout, a technique that randomly drops units and their connections from a neural network during training. 
+
+Current deep learning networks contain many layers, allowing them to learn complicated relationships from data. However, due to limitations in training data and the depth and complexity of modern data, some of the relationships are learned from overfitting and will not be seen in actual test data. 
+
+Ensemble methods work very well in many settings. These methods include combining several models, but is tough when considering large neural networks. Ensemble methods are the most effective when each individual model is unique from the rest, such as having a different architecture or being trained on different data. Given the current limits on computation power and smaller amounts of training data, having several unique networks is a difficult task. Instead, dropout simulates combining different neural networks together.
```
Comment (zhaoyue-zephyrus) [inline _summaries/srivastava2014dropout_1.md:16]: It is good to discuss the motivation of implicit ensemble behind dropout. But the current version seems a bit lengthy.

File: _summaries/srivastava2014dropout_1.md [hunk 2]
```diff
@@ -1,19 +1,114 @@
...
+* CIFAR-10 and CIFAR-100: Image classification sets
+* SVHN: Images of house numbers for classification to identify the digits
+* ImageNet: A collection of natural images for classification
+* Reuters-RCV1: A collection of Reuters articles
+* Alternative Splicing data set: RNA features for prediciting alternative gene splicing
+
+These sets were chosen to show that dropout is a general technique and can be used across multiple domains and tasks. 
+
+First, the paper examines the performance of dropout across the image data sets: MNIST, SVHN, CIFAR-10, CIFAR-100, and ImageNet. Across of the sets, state-of-the-art results were achieved. 
+
```
Comment (zhaoyue-zephyrus) [inline _summaries/srivastava2014dropout_1.md:59]: The summary on experiments is extensive. My suggestion would be make it a bit conciser and include some key observation. For example, some things I notice interesting are: (1) applying dropout at all layers beat applying dropout at fc layers only as well as no dropout (question: what is common practice nowadays? is this conclusion still holds? if not, what is the reason behind?) (2) The improvement of using dropout on Reuters-RCV1 is less significant than on vision and speech task. (question: what could be some reasonable explanation?)

File: _summaries/srivastava2014dropout_1.md [hunk 3]
```diff
@@ -1,19 +1,114 @@
...
+![Diagram of Removal](srivastava2014dropout_1b.png)
+
+In terms of seeing a description of the model, we can see the following:
+
+![Diagram of Removal](srivastava2014dropout_1e.png)
+
+Here, we can treat $$y$$ as the output of a unit, $$z$$ as the input of a layer, $$f$$ as the activation function, $$w$$ as the weights, and $$b$$ as the biases. These values can be calculated using the following equations: 
+
+![Diagram of Removal](srivastava2014dropout_1c.png)
```
Comment (zhaoyue-zephyrus) [inline _summaries/srivastava2014dropout_1.md:36]: Formula in latex might seem nicer.

**Review by zhaoyue-zephyrus — COMMENTED** [review]
Great job! The summary contains all important items of the paper. I've left some comments (and questions) in the reviews and you can change if you'd like. You don't need to write down your answer to the question but you might think it over.

**Comment by zhaoyue-zephyrus** [comment]
Also, please remove files `.DS_Store` and `_summaries/.DS_Store` using `git rm`.

File: _summaries/srivastava2014dropout_1.md (hunk 3 above)
Comment (patelojas) [reply _summaries/srivastava2014dropout_1.md:36]: I'm having trouble typing latex in the md document.

Comment (zhaoyue-zephyrus) [reply _summaries/srivastava2014dropout_1.md:34]: native markdown doesn't support latex but you can view the effect though `jekyll build`. But it's fine to use figure.

**Pull request merged**